clash-prelude/src/Clash/Signal.hs

```diff
@@ -241,9 +241,9 @@ module Clash.Signal
   , testFor
     -- * Type classes
     -- ** 'Eq'-like
-  , (.==.), (./=.)
+  , (.==.), (.==), (==.), (./=.), (./=), (/=.)
```

Comment:
It's further up in this file, but the new functions amongst `(.&&.), (&&.), (.&&), (.||.), (||.), (.||)` should also be imported.

Reply:
exported*